.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# The automate team will be requested for review on all opened PRs.
+* @globusonline/automate @ada-globus @joshbryan-globus @kurtmckee @rudyardrichter @jakeglobus
```

```suggestion
* @globus/automate @ada-globus @joshbryan-globus @kurtmckee @rudyardrichter @jakeglobus
```

This repo is in the `globus` org.

---

🙏